jobs/generation/MetaGenerator.groovy

```diff
@@ -240,28 +242,91 @@ repos.each { repoInfo ->
         folder(generatorFolder) {}
     }
 
+    // Create a view for all jobs in this folder that don't end with prtest
+    dashboardView("${projectFolder}/Official Builds") {
+        recurse()
+        jobs {
+            regex(/.*(?<!prtest)$/)
+        }
+
+        columns {
+            status()
+            weather()
+            name()
+            lastSuccess()
+            lastFailure()
+            lastDuration()
+        }
+
+        topPortlets {
+            jenkinsJobsList {
+                defaultName = "${projectFolder} jobs"
+            }
+        }
+
+        rightPortlets {
+            buildStatistics {
+                displayName 'Build Statistics'
+            }
+        }
+
+        def createPortletId = {
+            def random = new Random()
+            return "dashboard_portlet_${random.nextInt(32000)}"
```

Comment:
Does the ID matter?  This id will change every time the generator is run (which will be multiple times/day)